### PR TITLE
Prometheus Integartion supports ChangeFeed Metrics

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ Prometheus tracks the following metric data for your TiDB clusters.
 | tidbcloud_db_failed_queries_total | count | type: `planner:xxx\|executor:2345\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The total number of execution errors |
 | tidbcloud_db_connections | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | Current number of connections in your TiDB server |
 | tidbcloud_db_query_duration_seconds | histogram | sql_type: `Select\|Insert\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The duration histogram of statements |
+| tidbcloud_changefeed_latency | gauge | changefeed_id | The progress latency of data replication between the upstream and the downstream. It can also be referred to as replication lag |
+| tidbcloud_changefeed_Replica_rows | gauge | changefeed_id | The number of replica rows that ChangeFeed writes to the downstream per second |
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:87]: tidbcloud_changefeed_Replica_rows 中的 R 是小写吗？

Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:87]:
```suggestion
| tidbcloud_changefeed_replica_rows | gauge | changefeed_id | The number of replicated rows that a changefeed writes to the downstream per second |
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 2]
```diff
@@ -83,6 +83,8 @@ Prometheus tracks the following metric data for your TiDB clusters.
 | tidbcloud_db_failed_queries_total | count | type: `planner:xxx\|executor:2345\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The total number of execution errors |
 | tidbcloud_db_connections | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | Current number of connections in your TiDB server |
 | tidbcloud_db_query_duration_seconds | histogram | sql_type: `Select\|Insert\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The duration histogram of statements |
+| tidbcloud_changefeed_latency | gauge | changefeed_id | The progress latency of data replication between the upstream and the downstream. It can also be referred to as replication lag |
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:86]:
```suggestion
| tidbcloud_changefeed_latency | gauge | changefeed_id | The data replication lag between the upstream and the downstream of a changefeed |
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md (hunk 1 above)
Comment (lilyjazz) [reply tidb-cloud/monitor-prometheus-and-grafana-integration.md:87]: 是小写

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 3]
```diff
@@ -83,6 +83,8 @@ Prometheus tracks the following metric data for your TiDB clusters.
 | tidbcloud_db_failed_queries_total | count | type: `planner:xxx\|executor:2345\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The total number of execution errors |
 | tidbcloud_db_connections | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | Current number of connections in your TiDB server |
 | tidbcloud_db_query_duration_seconds | histogram | sql_type: `Select\|Insert\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The duration histogram of statements |
+| tidbcloud_changefeed_latency | gauge | changefeed_id | The data replication lag between the upstream and the downstream of a changefeed |
```
Comment (hfxsd) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:86]: 需要加上单位吗？比如毫秒、秒之类的。
```suggestion
| tidbcloud_changefeed_latency | gauge | changefeed_id | The data replication latency between the upstream and the downstream of a changefeed |
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md (hunk 3 above)
Comment (lilyjazz) [reply tidb-cloud/monitor-prometheus-and-grafana-integration.md:86]: 就是 s，但是面板会自动换算成 ms 之类的。

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json [hunk 4]
```diff
@@ -716,6 +718,208 @@
...
+          },
+          "editorMode": "code",
+          "expr": "tidbcloud_changefeed_Replica_rows",
+          "interval": "30s",
+          "legendFormat": "{{changefeed_id}}",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "Replica rows Per Sec",
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json:912]:
```suggestion
      "title": "Replicated Rows Per Sec",
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 5]
```diff
@@ -83,9 +83,17 @@ Prometheus tracks the following metric data for your TiDB clusters.
...
 | tidbcloud_node_cpu_capacity_cores | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU limit cores of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_used_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The used memory bytes of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_capacity_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The memory capacity bytes of TiDB/TiKV/TiFlash nodes |
+
+## FAQ
+
+1. Why is there a difference in the value of the same metric at the same time between Grafana and Cloud UI?
+
+Due to the differences in aggregation calculation logic between Grafana and the Cloud UI, there may be differences in the aggregated values displayed. Users can adjust the `mini step` configuration in Grafana to obtain more granular metric values.
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:99]:
```suggestion
- Why does the value of the same metric is different between Grafana and the Cloud console at the same time?

    The aggregation calculation logic is different between Grafana and TiDB Cloud, so the displayed aggregated values might differ. You can adjust the "mini step" configuration in Grafana to get more granular metric values.
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 6]
```diff
@@ -83,9 +83,17 @@ Prometheus tracks the following metric data for your TiDB clusters.
...
 | tidbcloud_node_cpu_seconds_total | count | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU usage of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_cpu_capacity_cores | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU limit cores of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_used_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The used memory bytes of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_capacity_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The memory capacity bytes of TiDB/TiKV/TiFlash nodes |
+
+## FAQ
+
+- Why does the value of the same metric is different between Grafana and the TiDB Cloud console at the same time?
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:97]:
```suggestion
- Why is the value of the same metric different between Grafana and the TiDB Cloud console at the same time?
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 7]
```diff
@@ -83,9 +83,17 @@ Prometheus tracks the following metric data for your TiDB clusters.
...
 | tidbcloud_node_cpu_seconds_total | count | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU usage of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_cpu_capacity_cores | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU limit cores of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_used_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The used memory bytes of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_capacity_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The memory capacity bytes of TiDB/TiKV/TiFlash nodes |
+
+## FAQ
+
+- Why is the value of the same metric different between Grafana and the TiDB Cloud console at the same time?
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:97]:
```suggestion
- Why does the same metric have different values on Grafana and the TiDB Cloud console at the same time?
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 8]
```diff
@@ -83,9 +83,17 @@ Prometheus tracks the following metric data for your TiDB clusters.
...
 | tidbcloud_node_cpu_capacity_cores | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU limit cores of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_used_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The used memory bytes of TiDB/TiKV/TiFlash nodes |
 | tidbcloud_node_memory_capacity_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The memory capacity bytes of TiDB/TiKV/TiFlash nodes |
+
+## FAQ
+
+- Why does the same metric have different values on Grafana and the TiDB Cloud console at the same time?
+
+    The aggregation calculation logic is different between Grafana and TiDB Cloud, so the displayed aggregated values might differ. You can adjust the `mini step` configuration in Grafana to get more granular metric values.
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:99]:
```suggestion
    The aggregation calculation logic is different between Grafana and TiDB Cloud, so the displayed aggregated values might differ. You can adjust the `mini step` configuration in Grafana to get more fine-grained metric values.
```

**Review by shhdgit — APPROVED** [review]

**Pull request merged**